### PR TITLE
DMP-692: Audio preview (UI Only)

### PR DESCRIPTION
**Pull request by deanbullock-hmcts**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/DMP-692

### CODE REVIEW

File: src/app/components/common/audio-player/audio-player.component.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<p>Loading Audio Preview........Please Wait</p>
```
Comment (davet1985) [inline src/app/components/common/audio-player/audio-player.component.html:1]: Maybe too many dots? 😂

File: src/app/components/common/loading/loading.component.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div class="loading-container">
```
Comment (davet1985) [inline src/app/components/common/loading/loading.component.html:1]: Can you rebase the loading changes out of your branch please.

File: src/app/components/hearing/events-and-audio/events-and-audio.component.ts [hunk 3]
```diff
@@ -83,6 +85,16 @@ export class EventsAndAudioComponent implements OnInit, OnChanges, OnDestroy {
     }
   }
 
+  onPreviewAudio(id: number) {
+    if (!this.isAudioInPreview(id)) {
+      this.audioInPreview = [...this.audioInPreview, id];
```
Comment (davet1985) [inline src/app/components/hearing/events-and-audio/events-and-audio.component.ts:90]: I think the scope of this ticket is to start the request to preview the audio, but not worry about the response.

**Review by davet1985 — CHANGES_REQUESTED** [review]
Couple of comments

File: src/app/components/common/audio-player/audio-player.component.html (hunk 1 above)
Comment (deanbullock-hmcts) [reply src/app/components/common/audio-player/audio-player.component.html:1]: Copy pasta from ticket 😂

File: src/app/components/common/loading/loading.component.html (hunk 2 above)
Comment (deanbullock-hmcts) [reply src/app/components/common/loading/loading.component.html:1]: I honestly don't know why your changes are in here 🤔

File: src/app/components/hearing/events-and-audio/events-and-audio.component.ts (hunk 3 above)
Comment (deanbullock-hmcts) [reply src/app/components/hearing/events-and-audio/events-and-audio.component.ts:90]: Ah, I just thought it was UI changes. I've handled the full thing in the next PR.

Comment (davet1985) [reply src/app/components/hearing/events-and-audio/events-and-audio.component.ts:90]: That's fine, as discussed

**Review by davet1985 — APPROVED** [review]
LGTM ⚡

**Pull request merged**